server/views/base.jade

```diff
@@ -7,6 +7,8 @@ html
     meta(name='viewport', content='width=device-width, initial-scale=1')
     meta(name='description', content='Project Description.')
     meta(name='author', content='gobble43')
+    script(src='https://maps.googleapis.com/maps/api/js?key=AIzaSyBdIp1gGNJfjr6krHhiaLFef-gUga8pESo')
```

Comment:
Let's try to avoid, or at least optimize, loading scripts in the headers. Ripple is only visible within the user-app page, right? Could we move these into a head scripts block within user-app.jade?